<launch>
    <arg name="vehicle_name" default="bluerov" />

    <include file="$(find bluerov_sim)/launch/gazebo_base.launch" />
    <group ns="$(arg vehicle_name)">

        <include file="$(find bluerov_sim)/launch/spawn_vehicle.launch">
            <arg name="z" value="-0.2" />
        </include>

        <node name="pid_node" pkg="bluerov_sim" type="altitude_hold.py" output="screen" />
        <node name="mixer" pkg="bluerov_sim" type="mixer.py" />
        <node name="lawnmower" pkg="bluerov_sim" type="LawnMow.py" />
        <node name="waypoint_server_node" pkg="bluerov_sim" type="WayPoint_server.py" />
    </group>

</launch>
